.github/ISSUE_TEMPLATE/inquiry-request.md

```diff
@@ -9,7 +9,7 @@ assignees: 'alxzak'
 
 **Please complete the following information:**
 
-Add any information or context about the inquiry here. What question do you need answer to?
+Add any information or context about the inquiry here. What question do you need an answer for?
```

Comment:
Alternatively: Add any information or context about the inquiry here as well as questions you need an answer for